python/paddle/static/io.py

```diff
@@ -784,7 +817,9 @@ def load_from_file(path):
 
 
 @static_only
-def load_inference_model(path_prefix, executor, **kwargs):
+def load_inference_model(
+    path_prefix: str | None, executor: Executor, **kwargs: Any
+) -> list[Program, list[str], list[Tensor]]:
```

Comment:
```suggestion
) -> list[Program | list[str] | list[Tensor]]:
```

`list` 只能有一个参数 ～ 其实，代码用 `tuple` 返回更好 ... ...